content/academy/data_collection_basics/api_scraping.md

```diff
@@ -152,29 +158,37 @@ const response = await gotScraping({
 })
 ```
 
-For our SoundCloud example, testing the endpoint from the previous section in a tool like [Postman](https://www.postman.com/) works perfectly, and returns the data we want; however, when the "`client_id`" parameter is removed, we receive a `401 Unauthorized` error. Luckily, the Client ID is the same for every user, which means that it is not tied to a session or an IP address (this is based on our own observations and tests). The big downfall is that the token being used by SoundCloud changes every few weeks, so it shouldn't be hardcoded. This case is actually quite common, and is not only seen with SoundCloud.
+For our SoundCloud example, testing the endpoint from the previous section in a tool like [Postman](./postman.md) works perfectly, and returns the data we want; however, when the `client_id` parameter is removed, we receive a _**401 Unauthorized**_ error. Luckily, the Client ID is the same for every user, which means that it is not tied to a session or an IP address (this is based on our own observations and tests). The big downfall is that the token being used by SoundCloud changes every few weeks, so it shouldn't be hardcoded. This case is actually quite common, and is not only seen with SoundCloud.
```

Comment:
Links to internal files work a little differently. But don't worry about it now, I'll change it anyway after the merge, to prepare for the upcoming Academy structure. Postman will go under Glossary.
![image](https://user-images.githubusercontent.com/23726914/158375477-7be1946e-e1d2-49fb-8b67-28d8d0b7ba9f.png)